teuthology/orchestra/cluster.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, remotes=None):
         """
         self.remotes = {}
         if remotes is not None:
-            for remote, roles in remotes:
+            for remote, roles in remotes.iteritems():
```

Comment:
agreed. remotes is a `list` not a `dict` here. please see `teuthology/orchestra/test/test_cluster.py` for examples. if i am wrong, and this is indeed a syntax error, could you add a test case for it?